suites/quincy/rgw/tier-1_rgw_multisite.yaml

```diff
@@ -295,21 +294,20 @@ tests:
             script-name: test_multisite_sync_policy.py
             config-file-name: test_multisite_sync_policy_extended.yaml
             verify-io-on-site: ["ceph-sec"]
-            timeout: 300
 
-  # - test:
-  #     name: bucket granular sync policy with allowed allowed semantic
-  #     desc: Test bucket granular sync policy with allowed allowed semantic
-  #     polarion-id: CEPH-83575868
-  #     module: sanity_rgw_multisite.py
-  #     comments: Known issue BZ-2153834 targeted to 8.0
-  #     clusters:
-  #       ceph-pri:
-  #         config:
-  #           script-name: test_multisite_bucket_granular_sync_policy.py
-  #           config-file-name: test_multisite_granular_bucket_sync_policy.yaml
-  #           verify-io-on-site: ["ceph-sec"]
-  #           timeout: 300
+  - test:
+      name: bucket granular sync policy with allowed allowed semantic
+      desc: Test bucket granular sync policy with allowed allowed semantic
+      polarion-id: CEPH-83575868
+      module: sanity_rgw_multisite.py
+      comments: Known issue BZ-2153834 targeted to 8.0
+      clusters:
+        ceph-pri:
+          config:
+            script-name: test_multisite_bucket_granular_sync_policy.py
+            config-file-name: test_multisite_granular_bucket_sync_policy.yaml
+            verify-io-on-site: ["ceph-sec"]
```

Comment:
Please add this as part of separate commit in this PR. Adding details of enablement.

Reply:
@psathyan , This was a working testcase. commented due to BZ.
So included this as a part of this pr